var/spack/repos/builtin/packages/py-libsonata/package.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyLibsonata(PythonPackage):
+    """SONATA files reader"""
+
+    homepage = "https://github.com/BlueBrain/libsonata"
+    url = "https://pypi.io/packages/source/l/libsonata/libsonata-0.0.2.tar.gz"
+
+    version('0.0.2', sha256='ea6655d8f1cb79262b5c5ad613386e88635ff7a4745cf7ef5643fd3c59be9a55', preferred=True)
+
```

Comment:
just curious : pypi tarball comes with pybind, highfive / all submodules?

Reply:
Yes, header-only dependencies `pybind11`, `highfive`, `fmt` are bundled into the tarball (only files necessary for compilation + license, for a total of +200K).
`hdf5` is expected to be available in the system.